ansible/cloud_providers/ec2_infrastructure_deployment.yml

```diff
@@ -58,6 +58,7 @@
         name: infra-ec2-template-create
       vars:
         aws_region_loop: "{{aws_region}}"
+      when: ec2_default_cloudformation | default(true) | bool
```

Comment:
that variable name looks wrong.
The role takes the cloudformation template from the *config* if it exists

The variable name suggests it would not do that if such a template exists.

Reply:
We can think of a variable to enable/disable the infra-ec2-template  roles all together, something more generic.

maybe  `infra_cloudformation_enabled` with a  default=true?

Reply:
I am trying to disable the entire deployment of cloudformation for an ec2 environment and use a cloudformation template that is located in my lab guide repo in `ansible/configs/ocp4-disconnected/post_infra.yml`. Is there a better variable name for "disable all default cloudformation in the sandbox"?

Reply:
Gotcha, I changed it to `cloudformation_enabled`